packages/grafana-ui/src/components/Cascader/Cascader.story.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean, object } from '@storybook/addon-knobs';
+import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
+import { Cascader } from './Cascader';
+
+const getKnobs = () => {
+  return {
+    disabled: boolean('Disabled', false),
+    text: text('Button Text', 'Click me!'),
+    options: object('Options', [
+      { label: 'A', value: 'A', children: [{ label: 'B', value: 'B' }, { label: 'C', value: 'C' }] },
+      { label: 'D', value: 'D' },
+    ]),
+  };
+};
+
+const CascaderStories = storiesOf('UI/Cascader', module);
```

Comment:
@kaydelaney FYI you don't need to use `storiesOf` anymore - you can use https://storybook.js.org/docs/formats/component-story-format/ instead